python/py-requests-cache/Portfile

```diff
@@ -4,8 +4,7 @@ PortSystem          1.0
 PortGroup           github 1.0
```

Comment:
since you removed the `github.setup` line and now will use PyPI to fetch the source, you can remove the PG as well